File: app/src/main/java/com/example/trigger/AboutActivity.java
package com.example.trigger;

import android.content.Context;
import android.content.SharedPreferences;
import android.preference.Preference;
import android.preference.PreferenceManager;
import android.preference.PreferenceActivity;
import android.os.Bundle;


public class AboutActivity extends PreferenceActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        addPreferencesFromResource(R.xml.about);

        Context context = this.getApplicationContext();
        SharedPreferences pref = PreferenceManager.getDefaultSharedPreferences(context);
        String version = Settings.getVersion();

        Preference versionPref = findPreference("prefVersion");
        versionPref.setSummary(version);
    }
}

File: app/src/main/java/app/trigger/AboutActivity.kt
package app.trigger

import android.preference.PreferenceActivity
import android.os.Bundle
import android.preference.PreferenceManager
import android.preference.Preference

class AboutActivity : PreferenceActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        addPreferencesFromResource(R.xml.about)
        val context = this.applicationContext
        val pref = PreferenceManager.getDefaultSharedPreferences(context)
        val version = Settings.app_version
        val versionPref = findPreference("prefVersion")
        versionPref.summary = version
    }
}
